### DISCUSSION TITLE
Abelljs icon added to Visual Studio Code icon pack!

### DISCUSSION THREAD

**Discussion by haaarshsingh** [abelljs/abell#110] [General]
The abelljs file icon has now been added to [this](https://marketplace.visualstudio.com/items?itemName=emmanuelbeziat.vscode-great-icons) icon pack! I opened a [issue](https://github.com/EmmanuelBeziat/vscode-great-icons/issues/188#issuecomment-751903574) for it and it has been added in the latest version :+1:

**Comment by haaarshsingh** [comment]
Hmmm, so this is strange.

![image](https://user-images.githubusercontent.com/69592270/103258140-f79f4e00-4961-11eb-8fd4-a71a6dd4cabf.png)

There is a small glitch where the icon is just blank... I have alerted about this, hopefully it gets fixed :+1:

**Reply by haaarshsingh** [reply]
It's fixed now!

![image](https://user-images.githubusercontent.com/69592270/103355029-2f4de900-4a7b-11eb-9af2-2ae26193e7b4.png)

Your files will have an awesome abelljs icon if you use the [VSCode great icons](https://marketplace.visualstudio.com/items?itemName=emmanuelbeziat.vscode-great-icons) :tada:

**Reply by saurabhdaware** [reply]
This is amazing! Thank you so much @harshhhdev

**Reply by prafulla-codes** [reply]
Looks cool

**Comment by MikeTeddyOmondi** [comment]
I suggest the icon be put in all major icons themes in VSCode
